CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 1.13
+* Tests now ensure that composer.lock is kept up to date. (GitHub #132)
+* The page title block is now properly placed in the Seven theme. (GitHub #190)
+* Twitter media entities now have automatically generated thumbnails.
+  (GitHub #203)
+* CI now uses database snapshots for update tests. (GitHub #201)
+* Fix multiversion regression of node revisions list. (Issue #2824633)
```

Comment:
s/multiversion/Multiversion. Also, can we link to the Multiversion issue that contains the patch?